Source/Core/DolphinQt/Debugger/PatchInstructionDialog.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt/Debugger/PatchInstructionDialog.h"
+
+#include <QDialogButtonBox>
+#include <QLabel>
+#include <QLineEdit>
+#include <QPushButton>
+#include <QVBoxLayout>
+
+#include "Common/GekkoDisassembler.h"
+
+PatchInstructionDialog::PatchInstructionDialog(QWidget* parent, u32 address, u32 value)
+    : QDialog(parent), m_address(address)
+{
+  setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+  setWindowModality(Qt::WindowModal);
+  setWindowTitle(tr("Instruction"));
+
+  CreateWidgets();
+  ConnectWidgets();
+
+  OnEditChanged();
+
+  m_input_edit->setText(QStringLiteral("%1").arg(value, 8, 16, QLatin1Char('0')));
```

Comment:
Shouldn't this be enough to trigger `OnEditChanged` so you don't need to explicitly call it?

Reply:
Yeah you're right.